recipes/safe-pysha3/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "safe-pysha3" %}
+{% set version = "1.0.4" %}
+{% set commit = "54be7e36b56c4734ca358c26b7734516cd748189" %}
+{% set sha256 = "292548b0e18d6043a468185d6ce27761204ebf4d694daef956b6ffb52a6871a8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
```

Comment:
Why aren't you referencing the tarball of the tag here?

Reply:
I do not understand, what is the `tarball of the tag`. I am not experienced recipe creator, these are my fist few, I tried to follow the example and other recipes. Another reviewer advised to use the github commit in order to pull in the LICENSE and the tests, rather than providing it with the recipe